frontroomsbot/cogs/tldr.py

```diff
@@ -0,0 +1,118 @@
+import discord
+from discord import app_commands, MessageType
+from bot import BackroomsBot
+from ._config import ConfigCog
+import google.generativeai as genai
+import re
+
+from consts import GEMINI_TOKEN
+
+USER_RE = re.compile(r"<@\d+>")
+
+
+class TldrCog(ConfigCog):
```

Comment:
Change this to inherit from commands.Cog, ConfigCog would make it appear in configurable menu even tho it has nothing to be configured resulting in an error

Reply:
fixed in the last commit